### PR TITLE
Feature/change date format

### PR DESCRIPTION
**Pull request by danaronson**

### CODE REVIEW

File: contacts.py [hunk 1]
```diff
@@ -216,9 +214,9 @@ def scan_status(self, data, args):
         ret = {}
         if since:
             ret['since'] = since
-            since = int(unix_time(datetime.datetime.strptime(since, "%Y%m%d%H%M")))
+            since = int(unix_time(datetime.datetime.fromisoformat('2007-04-05T14:30Z'.replace("Z", "+00:00"))))
```
Comment (mitra42) [inline contacts.py:217]: What is this line - looks like a hard-coded date to me ?

File: contacts.py [hunk 2]
```diff
@@ -250,10 +248,10 @@ def sync(self, data, args):
         if since:
             since = since[0].decode()
         else:
-            since = "197001010000"
+            since = "1970-01-01T01:01Z"
 
-        since = int(unix_time(datetime.datetime.strptime(since,
-                                                         "%Y%m%d%H%M")))
+        since = int(unix_time(datetime.datetime.fromisoformat('2007-04-05T14:30Z'.replace("Z", "+00:00"))))
```
Comment (mitra42) [inline contacts.py:253]: Another hard coded date

File: tests/conftest.py [hunk 3]
```diff
@@ -78,7 +78,7 @@ def reset(self):
                 os.unlink(file_path)
             elif os.path.isdir(file_path):
                 shutil.rmtree(file_path)
-        #os.kill(60328, SIGUSR1)
+        #os.kill(76617, SIGUSR1)
```
Comment (mitra42) [inline tests/conftest.py:81]: What is this?

File: tests/test_scan_status.py [hunk 4]
```diff
@@ -9,7 +9,7 @@ def test_scan_status(server, data):
     data = [{"id":contact_id}]
     server.send_status(contacts = data, status = 2)
     expected = [ { 'id': contact_id, 'status': 2 }]
-    resp = server.scan_status(contact_prefixes = [prefix])
+    resp = server.scan_status(contact_prefixes = [prefix], since = "2007-04-05T14:30Z")
```
Comment (mitra42) [inline tests/test_scan_status.py:12]: And another hard coded date - though maybe fine in a test.

File: contacts.py (hunk 1 above)
Comment (danaronson) [reply contacts.py:217]: woops.

File: tests/conftest.py (hunk 3 above)
Comment (danaronson) [reply tests/conftest.py:81]: ignore, it's commented out.  it's support for being able to run the server not as a subprocess for some of my debugging.

**Pull request merged**